src/rust/client-executor/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "client-executor"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+futures = "0.3.21"
+pin-project = "1.0.11"
+recloser = "1.0.0"
+tokio = { version = "1.20.0", features = ["full"] }
```

Comment:
rest of our codebase is 1.17

Reply:
`version = "1.X"` will choose any version above `1.x`. So if we specify 1.17 elsewhere and 1.20 here we'll end up with 1.20 everywhere, and it'll be a lockfile failure otherwise. You can confirm this in the lockfile - you'll see 1.20 is the version in use for tokio.